### ISSUE TITLE
vendoring breaks caddy's dns plugin

### ISSUE THREAD

**Issue by miekg** [coredns/coredns#696]
from @mholt:

"CoreDNS doesn't "plug in" to Caddy anymore (including from the Caddy
website), and I think it has to do with vendoring the mholt/caddy repo,
because it registers to the vendored (copied) caddy core, instead of the
"main" one."

**Comment by miekg** [comment]
looks like you can just edit Gopkg.toml:
~~~
# "ignored" lists a set of packages (not projects) that are ignored when
## dep statically analyzes source code. Ignored packages can be in this project,
## or in a dependency.
# ignored = ["github.com/user/project/badpkg"]
~~~

**Comment by miekg** [comment]
~~~
diff --git a/Gopkg.toml b/Gopkg.toml
index 6b7a32ba..eb44883f 100644
--- a/Gopkg.toml
+++ b/Gopkg.toml
@@ -1,4 +1,3 @@
-
 ## Gopkg.toml example (these lines may be deleted)
 
 ## "required" lists a set of packages (not projects) that must be included in
@@ -10,7 +9,8 @@
 ## "ignored" lists a set of packages (not projects) that are ignored when
 ## dep statically analyzes source code. Ignored packages can be in this project
 ## or in a dependency.
-# ignored = ["github.com/user/project/badpkg"]
+# See: https://github.com/coredns/coredns/issues/696
+ignored = ["github.com/mholt/caddy"]
~~~
running `dep ensure` and `dep prune` again and seeing what happens.

**Comment by miekg** [comment]
This does not work...

**Comment by miekg** [comment]
see #700  I was holding it wrong

**Comment by miekg** [comment]
closed via #700
